src/simulation/environment/dentonFluxModel/_UnitTest/test_dentonFluxModel.py

```diff
@@ -166,26 +166,32 @@ def dentonFluxModelTestFunction(show_plots, param1_Kp, param2_LT, param3_z, para
     trueIonFluxData = np.array([0.0] * messaging.MAX_PLASMA_FLUX_SIZE)
     with open(filepath, 'r') as file:
         rows = np.loadtxt(file, delimiter=",", unpack=False)
+        # true flux data provided by Denton is in Units of [cm^-2 s^-1 sr^-2 eV^-1], but DentonFluxModel converts it to
+        # [m^-2 s^-1 sr^-2 eV^-1]. Need to multiply by 1e4
         trueEnergyData[0:module.numOutputEnergies] = rows[0]
-        trueElectronFluxData[0:module.numOutputEnergies] = 10.**(rows[1])
-        trueIonFluxData[0:module.numOutputEnergies] = 10.**(rows[2])
+        trueElectronFluxData[0:module.numOutputEnergies] = 10.**(rows[1]) * 1e4
+        trueIonFluxData[0:module.numOutputEnergies] = 10.**(rows[2]) * 1e4
+
+    print(trueElectronFluxData)
```

Comment:
This print statement should be removed, or at least commented out?

Reply:
Removed it.